components/compositing/compositor.rs

```diff
@@ -1465,15 +1489,17 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
     /// scrolling to the applicable scroll node under that point. If a scroll was
```

Comment:
GitHub is not allowing me to add the comment on the exact line, but the documentation above this line needs to be updated to use `DevicePoint` instead of `WorldPoint`.

```
    /// Perform a hit test at the given [`WorldPoint`]
 ```